frontend/src/components/common/ProductDisplay/ProductList/Product/Product.scss

```diff
@@ -0,0 +1,79 @@
+.product {
+  background-color: transparent;
+
+  &__container {
+    position: relative;
+    min-width: 256px;
+    min-height: 256px;
+    overflow: hidden;
+  }
+
+  &__image {
+    width: 256px;
+    height: 256px;
+    object-fit: cover;
+    background-color: #43434375;
+    opacity: 0.9;
+    transition: transform 0.5s;
+  }
+
+  &__info {
+    position: absolute;
+    top: 0;
+    left: 0;
+    background-color: transparent;
+
+    &-name {
+      font-size: 18px;
+      font-weight: 700;
+      line-height: 1.5rem;
+      background-color: black;
+      padding: 16px 24px;
+    }
+
+    &-price {
+      font-size: 14px;
```

Comment:
we should write CSS follow this order:
![image](https://user-images.githubusercontent.com/69567295/187177048-96ad6ab4-4f72-4c26-b7d5-88885c2221b4.png)